### ISSUE TITLE
Bug: Object properties displaying C string types

### ISSUE THREAD

**Issue by prescience-data** [fullstack-build/tslog#188]
**Describe the bug**
When logging string values that have not had their underlying structure stripped, tslog exposes the object form rather than the string form. 

 

```js
    // Normal string:
    BUILT_AT: 'Thu, 01 Dec 2022 00:51:02 GMT',
    // String, but with C string structure data expose during serialization:
    RELEASE_CHANNEL: {
      '0': 'd',
      '1': 'e',
      '2': 'v',
      '3': 'e',
      '4': 'l',
      '5': 'o',
      '6': 'p',
      '7': 'm',
      '8': 'e',
      '9': 'n',
      '10': 't'
    }
```
**To Reproduce**

This is not the cause of the bug but this is an example scenario I've encountered.
 
1. Create two environment files with overlapping values.
```env
# .env
FOO=foo
BAR=bar
```

```env
# .env.defaults
FOO=foo
```

1. Create `docker-compose.yml` file with multiple local environment variables files specified.
```yml
  env_file:
    - .env.defaults
    - .env
```
3. Start a simple Node app using `tslog` from within container.
4. Log `process.env` with 
  a) Native console: `console.log("console: ", process.env)`, and;
  b) TSLog: `logger.info("tslog: ", process.env)` 

5. See difference for any overlapping values loaded by Docker.

```shell
console: {
  FOO: "foo",
  BAR: "bar"
}

```

```shell
tslog: {
  FOO: {
      '0': 'f',
      '1': 'o',
      '2': 'o'
    },
  BAR: "bar"
}

```

**Expected behavior**
Console output for logged object should not expose underlying string strucutre.
 
**Additional context**
For reference, both `typeof` return as `string`. 

**Node.js Version**
e.g. 18.x

**OS incl. Version**
e.g. Unbuntu 22.04

**Comment by prescience-data** [comment]
duplicate of https://github.com/fullstack-build/tslog/issues/184

**Comment by terehov** [comment]
@prescience-data Do I understand it right, that this issue has been fixed with `V4.4.4`?

**Comment by prescience-data** [comment]
Sure is, thanks!
